packages/react-broadcast/src/components/index.ts

```diff
@@ -1,3 +1,4 @@
 export { default as ToggleFeature } from './toggle-feature';
 export { default as ConfigureFlopFlip } from './configure';
 export { default as ReconfigureFlopFlip } from './reconfigure';
+export { default as FlopflipTestProvider } from './test-provider';
```

Comment:
Nit: to keep the name consistent?

```suggestion
export { default as TestProviderFlopFlip } from './test-provider';
```

Reply:
Sounds odd but consistent :)